### ISSUE TITLE
Make Org Export/Publish work out of the box

### ISSUE THREAD

**Issue by salutis** [org-roam/org-roam#2131]
I decided to try Org Roam again and found that everything works "out of the box" except exporting/publishing.  I found [1] as a suggested workaround and it solved the problem.  So, both Org Export and Org Publish works now.  Then, I do not understand the logic.  Do we expect everyone to find this (rather large) procedure and add it to their configuration file?  If so, then why not bundle it with the Org Roam package and call it `org-roam-export-insinuate` or something similar?

What do I miss?

[1]:

```emacs-lisp
(defun org-html--reference (datum info &optional named-only)
  (let* ((type (org-element-type datum))
	 (user-label
	  (org-element-property
	   (pcase type
	     ((or `headline `inlinetask) :CUSTOM_ID)
	     ((or `radio-target `target) :value)
	     (_ :name))
	   datum))
         (user-label
          (or user-label
              (when-let ((path (org-element-property :ID datum)))
                (concat "ID-" path)))))
    (cond
     ((and user-label
	   (or (plist-get info :html-prefer-user-labels)
	       (memq type '(headline inlinetask))))
      user-label)
     ((and named-only
	   (not (memq type '(headline inlinetask radio-target target)))
	   (not user-label))
      nil)
     (t
      (org-export-get-reference datum info)))))
```

_edit: wording and syntax highlighting_

**Comment by jethrokuan** [comment]
Addressed with #2138.

**Comment by salutis** [comment]
@jethrokuan Thank you!
